package.json

```diff
@@ -52,6 +52,7 @@
     "stylelint-config-standard": "^18.2.0"
   },
   "peerDependencies": {
+    "@ovh-ux/ng-ovh-mondial-relay": "^5.0.0",
```

Comment:
Since a 💥 **Breaking Change** will be introduced, we have to upgrade this version as soon as the component will be release.

ref: https://github.com/ovh-ux/ng-ovh-mondial-relay/pull/20#discussion_r277536260

Reply:
A new release of `@ovh-ux/ng-ovh-mondial-relay` has been published.

ref: https://github.com/ovh-ux/ng-ovh-mondial-relay/blob/master/CHANGELOG.md#600-2019-04-23

Reply:
thanks you @antleblanc, the dependancy has been updated, since the release 6.0.0 of `@ovh-ux/ng-ovh-mondial-relay`